sources/models/src/aws-k8s-1.15/mod.rs

```diff
@@ -16,6 +16,7 @@ struct Settings {
     kubernetes: KubernetesSettings,
     updates: UpdatesSettings,
     host_containers: HashMap<Identifier, ContainerImage>,
+    bootstrap_containers: HashMap<Identifier, BootstrapContainer>,
```

Comment:
If we're using the map key as the container ID, we should probably create a new type that validates the length.  containerd enforces a 76-character maximum on the container ID.

Reply:
I'll make this update just for bootstrap containers, I don't want to break existing customers for normal host containers.

Reply:
I added a validation to `Identifier` to check for the length